settled_handler.go

```diff
@@ -64,22 +65,27 @@ func (p *SettledHandler) WaitForInvoiceSettled(ctx context.Context,
 	// Check database to see if invoice was already settled.
 	invoice, _, err := p.persister.Get(ctx, hash)
 	if err != nil {
-		return err
+		return types.StatusUnknown, err
```

Comment:
For enums that don't matter, I usually return `0, err`. To make it clear that it really does not matter what the primary return value is.

Reply:
Never thought of that, nice one, will go for `"", err`